standards/04-community.md

```diff
@@ -0,0 +1,431 @@
+                  <a href="https://github.com/user">
+          <img
+            class="avatar__photo avatar__photo--xl margin-bottom--md"
+            src="https://github.com/user.png"
+          />
+          </a>
+          <div class="avatar__intro">
+            <h3 class="avatar__name">Team Memeber</h3>
+          </div>
+        </div>
+    </div>
+  </div>
+
+
+## Honorary Mentions
+
+- **Name**: [Acknowledgement]
+- **Name**: [Acknowledgement]
+- **Name**: [Acknowledgement]
+- **Name**: [Acknowledgement]
+
+``````
+
+</details>
+
+## Funding Page
+
+A transparent overview of the financial support and organizations that make the project possible. This page acknowledges funding sources, grants, and additional support while providing information about project sustainability and institutional backing.
+
+<details>
+<summary>**Click here for our templated funding page**</summary>
+
+``````
+# Funding
+
+We gratefully acknowledge the financial support that has made this project possible. Our work has been funded by the following organizations and grants:
+
+### [Funding Source]
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+  - Grant Number: [#U24CA253529]
+  - Project Title: "Project Title" 
+  - Funding Period: [YYYY-YYYY]
+
+### [Funding Source]
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+  - Grant Number: [XXX-XXXX]
+  - Project Title: "Project Title" 
+  - Funding Period: [YYYY-YYYY]
+
+
+### [Funding Source]
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+  - Grant Number: [XXX-XXXX]
+  - Project Title: "Project Title" 
+  - Funding Period: [YYYY-YYYY]
+
+## Additional Support
+
+We also thank the following organizations for their contributions:
+
+- [Supporting Organization 1] - [Type of Support]
+- [Supporting Organization 2] - [Type of Support]
+- [Supporting Organization 3] - [Type of Support]
+
+For more information about our funding or to discuss potential collaborations, please email us at contact@overture.bio.
+
+**Last updated:** mm-dd-yy
+``````
+
+</details>
+
+
+## Code of Conduct Page
+
+A clear set of guidelines establishing community behavior standards and expectations. This document ensures a respectful, inclusive, and harassment-free environment for all community members while outlining enforcement procedures for maintaining these standards.
+
+<details>
+<summary>**Click here for our code of conduct page**</summary>
+
+``````
+# Code of Conduct
```

Comment:
isn't this the same one in the code of conduct file? should this be a link to that instead of duplicating the text?

Reply:
Idea is having them templated and available for reproducibility, if you looked form the doc site its a page that just has all the relevant community resources https://main--overturedev.netlify.app/docs/standards/communityin an easy to copy paste manner